src/views/Portal/components/ApplicationForm/ShortAnswer/index.view.tsx

```diff
@@ -13,12 +14,13 @@ const ShortAnswerPage: React.FC = () => {
           <div className='short-answer-page-container__field'>
             <TextBox
               name='Why do you want to attend CruzHacks?'
-              handleChange={(e: React.ChangeEvent<HTMLInputElement>) =>
+              handleChange={(e: React.ChangeEvent<HTMLInputElement>) => {
                 setShortAnswerFormData(prev => ({
                   ...prev,
                   whyCruzHacks: e.target.value,
                 }))
-              }
+                setNewChanges()
+              }}
```

Comment:
I think functionally it works but, it's a little inconsistent from my testing. It seems like this will only run onChange of the textContainer used. I think `setNewChanges` would need to be set on every input component.

Reply:
Ooops, that’s my bad. I thought this was the `handleChange` function like the other ones. Good catch!